draft-ietf-quic-transport.md

```diff
@@ -2830,13 +2830,15 @@ valid frames? -->
 
 ### Sending ACK Frames
 
-<!-- TODO: Re-read this section for flow and redundancy. -->
-
-To avoid creating an indefinite feedback loop, an endpoint MUST NOT send an ACK
-frame in response to a packet containing only ACK or PADDING frames, even if
-there are packet gaps which precede the received packet.  The endpoint MUST
-however acknowledge packets containing only ACK or PADDING frames when sending
-ACK frames in response to other packets.
+ACK frames are sent to acknowledge received packets and indicate the frames
+within them has been processed. An endpoint MUST NOT send more than one packet
+containing only an ACK frame per received packet that contains frames other
+than ACK and PADDING frames. Specifically, to avoid creating an indefinite
```

Comment:
Can we simplify and say: no more than one packet containing only ACK and PADDING frames per packet received?  And then: no packets at all in response to a packet containing only ACK and PADDING [, even if gaps...].

Reply:
Updated, PTAL